### DISCUSSION TITLE
Next TSC Meeting - Thu 2024-11-06 @ 1300 UTC

### DISCUSSION THREAD

**Discussion by planetf1** [pq-code-package/tsc#112] [Announcements]
The next PQCP TSC meeting is on 2024-11-06 at 13:00 UTC.

This is the same time in UTC as usual (the meeting is scheduled in UTC), however given that many of you will have changed from daylight savings to winter time locally, it may be an hour earlier than expected. ( A particular pain point for those in the US/Canada )

During the meeting we can discuss if the time is ok for future meetings, or we want to move later.

I’ll create a PR with the meeting agenda shortly, and post in the discussions.

Nigel.

**Comment by franziskuskiefer** [comment]
Unfortunately I have other standing meetings in the 13:00 UTC time slot and won't be able to make it.

**Reply by planetf1** [reply]
Is that a one-off, or ongoing. If so I'll take that as a request for us to revisit the time, perhaps changing to follow uk/europe summer time patterns.
